src/aap_eda/core/utils/credentials.py

```diff
@@ -236,13 +236,9 @@ def validate_schema(schema: dict) -> list[str]:
 
 def validate_injectors(schema: dict, injectors: dict) -> dict:
     errors = []
-    context = _default_context(schema)
 
     if not isinstance(injectors, dict):
-        errors.append(
-            "Injectors must be a dict type and defines keys"
-            f" in {SUPPORTED_KEYS_IN_INJECTORS}"
-        )
+        errors.append("Injectors must be in dict format")
```

Comment:
This error will be shown to UI/API users, but dict is a python term.

Reply:
OK, let me change `dict` into `Key-Value pair`